ADD_SAMPLE_INTO_DEV_KIT.md

```diff
@@ -12,7 +12,10 @@ For new samples, use `dev` as the target branch for pull requests. The `dev` bra
 
 ## Testing
 
-TBD
+1. Download the [test package](./assets/dev-kit-test-package.vsix).
+2. Set the environment variable `OTK_SAMPLE_CONFIG_BRANCH` to `dev` or other branch with a valid [sample-config.json](./.config/sample-config.json) of this repo on your devbox. Setting this env variable will force sample gallery to consume the config of dedicated branch.
+3. Uninstall the installed Office Add-ins Development Kit and install the `dev-kit-test-package.vsix` manually in VSCode.
+4. Open Dev Kit. Select `View Samples` to view sample gallery and do testing. 
```

Comment:
Should this refer to the test package extension instead of the Dev Kit instead?

Reply:
Fixed